### ISSUE TITLE
Suggestion: implement scale force reconnect similar to mimojacafe

### ISSUE THREAD

**Issue by bbender716** [Damian-AU/DSx#5]
I've noticed that in the MimojaCafe skin, using its force reconnect function has been much more successful on average than the ability to force reconnect on DSX. It may be placebo, but just something I thought I'd offer as feedback. Love your work!

**Comment by Damian-AU** [comment]
I can't see how that would be possible, they both use the same core procedure `ble_connect_to_scale`

**Comment by bbender716** [comment]
Interesting. Maybe it's only a UX "issue" then where Mimoja indicates it's
attempting to reconnect when you push it. Sorry!

**Comment by Damian-AU** [comment]
I never power off my Skale, nor do I ever see dropped connects, so I just did a short test by power cycling my Skale to cause a disconnect. I had to wait for the app to recognise the scale is not responding, at which time DSx flashed "reconnect" + there is the android toast popup telling you the scale is disconnected, when I tap to reconnect it connected immediately, I repeated it 8x. 
I also switched to Mimoji to confirm the same process, similar to DSx it wouldn't reconnect until it recognises the disconnection.
So from my limited testing, they appear to work the same.
